schema.json

```diff
@@ -774,26 +764,6 @@
     "type": "STRING",
     "mode": "NULLABLE"
   },
-  {
-    "name": "inflationFactor",
-    "type": "FLOAT",
-    "mode": "NULLABLE"
-  },
-  {
-    "name": "inflationRate",
-    "type": "FLOAT",
-    "mode": "NULLABLE"
-  },
-  {
-    "name": "updatePeriod",
-    "type": "INTEGER",
-    "mode": "NULLABLE"
-  },
-  {
-    "name": "factorLastUpdated",
-    "type": "INTEGER",
-    "mode": "NULLABLE"
-  },
```

Comment:
Are these not being used?

Reply:
They were duplicated in the schema, once as string and once as integer or float, with data in the string columns. No idea what happened there

Reply:
Ouch! Thanks for fixing :)